test/testinput/bufrtest_python_to_ioda.py

```diff
@@ -1,75 +0,0 @@
-# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
-#
-# This software is licensed under the terms of the Apache Licence Version 2.0
-# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
-
-import numpy as np
-
-import bufr
-from pyioda import ioda_obs_space as ioda_ospace
```

Comment:
This test requires IODA and it is not available here.  We should remove this test

Reply:
It is done already!!  Thanks.